tools/run_tests/xds_k8s_test_driver/tests/authz_test.py

```diff
@@ -253,9 +253,10 @@ def test_tls_allow(self) -> None:
             self.configure_and_assert(test_client, None,
                                       grpc.StatusCode.PERMISSION_DENIED)
 
-        with self.subTest('03_principal_present'):
-            self.configure_and_assert(test_client, 'principal-present',
-                                      grpc.StatusCode.PERMISSION_DENIED)
+        # b/202058316
```

Comment:
How about:

```
# TODO(b/202058316) enable this sub test when TD supports wildcard principal
```

Reply:
Well, it supports wildcard principal, but not quite right. The details are in the bug and it should be resolved in a few days. It certainly will be resolved before GA, so I'm not too concerned about making sure the comment is super-good.